### ISSUE TITLE
Unchecking create subfolder option still creates subfolder on adding a torrent

### ISSUE THREAD

**Issue by hansari97** [WDaan/VueTorrent#584]
pretty much what the heading says. if possible add the option to change content layout: original, create subfolder, don't create subfolder like the default qbit ui
Thanks!

**Comment by Larsluph** [comment]
This box comes from the `root_folder` parameter from the route to add torrents [source](https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-(qBittorrent-4.1)#add-new-torrent)

**Comment by flashlab** [comment]
I met the same situation. the create subfolder option seems not working.

**Comment by Larsluph** [comment]
The problem is that the documentation isn't very clear about its true behaviour. From what I've read it seems that this option creates a root folder when a torrent have multiple files without a root folder.

This example will create a root folder named `torrentname`:
```
torrentname.torrent
  - file1
  - file2
  - file3
```

But this one won't
```
torrentname.torrent
  - file1
```
And this one either
```
torrentname.torrent
  - folder
    - file1
    - file2
```

**Comment by Larsluph** [comment]
If you really want to check that this checkbox is working:
- open VueTorrent on a desktop
- press F12 to open DevTools
- Go in the "Network" tab
- add a torrent
You should see a request named `createFile`, if you click on it and go in the `payload` tab, you should see a `root_folder` parameter with the value you set in the add modal.

**Comment by flashlab** [comment]
@Larsluph You're right. We misunderstood the meaning of such option. The behavior we want is to remove root folder defined by the torrent file, just like the "folder" in your demo3. So I should submit a feature request in qBittorrent itself, but not here.

Also I tested `root_folder` param in local PC, and vuetorrent just worked fine. Thank you for your explanation ;)

**Comment by jathek** [comment]
> So I should submit a feature request in qBittorrent itself, but not here

qBittorrent already has this feature, unless I'm misunderstading. That's what the `content layout: original, create subfolder, don't create subfolder` options are in the stock add torrent dialog. VueTorrent doesn't have this option, so I have to switch out of VueTorrent if I want to add a torrent but remove the default root folder. If I understand the API documentation, setting `root_folder` to false should do that. By default `root_folder` is unset, which uses whatever the torrent has by default.

**Comment by Larsluph** [comment]
Okay so I digged into the API source and it seems like the `root_folder` parameter isn't used anymore (since 1d5dc283fee8e7cf6c385140de13a9a6421f46bc, committed the 10th december 2020 😱) even though the WebUI API wiki still mentions it.
I will first update qb's wiki and then update this "new" behavior in VueTorrent.

I just tested on the current API and I can confirm that `root_folder` has no effects and that the `contentLayout` parameter works like mentioned above.
